### PR TITLE
Install binutils from GCC5

### PR DESCRIPTION
**Pull request by uilianries**
Related to https://github.com/conan-io/conan-docker-tools/pull/370/files#r803628721

- Why profile file? 
  - To generate that lockfile we run:

    conan lock create -r conancenter -pr profile --lockfile-out=conan.lock conanfile.txt

- [ ] Refer to the issue that supports this Pull Request.
- [ ] If the issue has missing info, explain the purpose/use case/pain/need that covers this Pull Request.
- [ ] I've read the [Contributing guide](https://github.com/conan-io/conan-docker-tools/blob/master/.github/CONTRIBUTING.md).
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008) style guides for Python code.
- [ ] I've followed the [Best Practices](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/) guides for Dockerfile.

### CODE REVIEW

File: modern/base/profile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+[settings]
+os=Linux
+os_build=Linux
+arch=x86_64
+arch_build=x86_64
+compiler=gcc
+compiler.version=5
+compiler.libcxx=libstdc++
```
Comment (jgsogo) [inline modern/base/profile:8]: old ABI?

Comment (uilianries) [reply modern/base/profile:8]: let me update to libstdc++11

Comment (uilianries) [reply modern/base/profile]: Done

**Review by jgsogo — APPROVED** [review]
IMHO, better to use GCC5 binaries, are they are linked using the lowest stdlibc++ version... just in case we add some C++ application in the future.

wdyt, @SSE4 ? is my understanding right?

**Comment by SSE4** [comment]
I think it should be irrelevant, as bintutils are pure C (at least, ld and gas) AFAIK and shouldn't introduce a link to libstdc++.

```
conan@08023348d030:~$ ldd $(which as)
	linux-vdso.so.1 =>  (0x00007ffdb5daa000)
	libopcodes-2.26.1-system.so => /usr/lib/x86_64-linux-gnu/libopcodes-2.26.1-system.so (0x00007f41bc865000)
	libbfd-2.26.1-system.so => /usr/lib/x86_64-linux-gnu/libbfd-2.26.1-system.so (0x00007f41bc520000)
	libz.so.1 => /lib/x86_64-linux-gnu/libz.so.1 (0x00007f41bc306000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f41bbf3c000)
	libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007f41bbd38000)
	/lib64/ld-linux-x86-64.so.2 (0x00007f41bcc0b000)
conan@08023348d030:~$ ldd $(which ld)
	linux-vdso.so.1 =>  (0x00007ffc165e7000)
	libbfd-2.26.1-system.so => /usr/lib/x86_64-linux-gnu/libbfd-2.26.1-system.so (0x00007f302803d000)
	libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007f3027e39000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f3027a6f000)
	libz.so.1 => /lib/x86_64-linux-gnu/libz.so.1 (0x00007f3027855000)
	/lib64/ld-linux-x86-64.so.2 (0x00007f3028382000)
```
but for some foreseen future it should be fine

File: modern/base/conan.lock [hunk 2]
```diff
@@ -40,13 +40,13 @@
    "4": {
     "ref": "termcap/1.3.1#a75d9f61bf0d1899b64967958dfa6538",
     "options": "fPIC=True\nshared=False",
-    "package_id": "dfbe50feef7f3c6223a476cd5aeadb687084a646",
-    "prev": "383d0c76a636c07aa78d0065f1e0a6f8",
+    "package_id": "76f87539fc90ff313e0b3182641a9bb558a717d2",
+    "prev": "34552acd6624780a5f6251750f40d9c5",
```
Comment (SSE4) [inline modern/base/conan.lock:44]: why package ID has changed?

Comment (uilianries) [reply modern/base/conan.lock:44]: Compiler version, from GCC11 to GCC5.

Comment (SSE4) [reply modern/base/conan.lock:44]: okay, so it's not removed in `package_id`...

**Review by SSE4 — APPROVED** [review]

**Pull request merged**